src/server/pegasus_server_impl.cpp

```diff
@@ -1490,6 +1496,16 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
         this, _read_hotkey_collector, _write_hotkey_collector);
     _server_write = dsn::make_unique<pegasus_server_write>(this, _verbose_log);
 
+    ::dsn::tasking::enqueue_timer(LPC_ANALYZE_HOTKEY,
```

Comment:
Where is the definition of `LPC_ANALYZE_HOTKEY`?

Reply:
https://github.com/XiaoMi/rdsn/blob/7601254a6ed140826276da3d35175530fea994f6/include/dsn/dist/replication/replication.codes.h#L74